content/_layouts/base.njk

```diff
@@ -59,16 +59,8 @@
     <script src="https://use.typekit.net/slx1xnq.js"></script>
     <script>try{Typekit.load({ async: true });}catch(e){}</script>
 
-    {% if site.google_analytics_id and env.ELEVENTY_ENV === 'production' -%}
-      <!-- Global site tag (gtag.js) - Google Analytics -->
-      <script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics_id }}"></script>
-      <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag(){dataLayer.push(arguments);}
-        gtag('js', new Date());
-
-        gtag('config', '{{ site.google_analytics_id }}');
-      </script>
+    {% if env.ELEVENTY_ENV === 'production' -%}
+      <script defer data-domain="oddbird.net" src="https://plausible.io/js/plausible.js"></script>
```

Comment:
Looks like this enables tracking on PR previews as well, which is convenient, but maybe it muddies the real tracking data?

Reply:
Hm, interesting. We might want to use the `CONTEXT` [env var](https://docs.netlify.com/configure-builds/environment-variables/#build-metadata) and [pass that through](https://docs.netlify.com/integrations/frameworks/environment-variables/#netlify-variables) to our 11ty build command to opt out for deploy previews? I don't think we want to include them (or local dev) unless there's a quick way to filter them out.

Reply:
I changed the conditional to read from `CONTEXT` in 82c1b81e8d4b103f2d59f74616969a6b55da2d43 and it seems to be working as expected. This was also the only instance of `ELEVENTY_ENV` in the whole codebase so we can go ahead and remove it, unless it's used by Eleventy itself 🤷‍♂️